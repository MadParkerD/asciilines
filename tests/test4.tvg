10 10
a 0 0 h 10
b 1 0 h 10
c 2 0 h 10
d 3 0 h 10
e 4 0 h 10
f 5 0 h 10
g 6 0 h 10
h 7 0 h 10
i 8 0 h 10
j 9 0 h 10
a 0 0 v 10
b 1 0 v 10
c 2 0 v 10
d 3 0 v 10
e 4 0 v 10
f 5 0 v 10
g 6 0 v 10
h 7 0 v 10
i 8 0 v 10
j 9 0 v 10
k 0 1 v 10
l 1 1 v 10
m 2 1 v 10
n 3 1 v 10
o 5 1 v 10
p 6 1 v 10
q 7 1 v 10
r 8 1 v 10
s 9 1 v 10
t 0 2 v 10
u 1 2 v 10
v 2 2 v 10
w 3 2 v 10
x 4 2 v 10
y 5 2 v 10
z 6 2 v 10
