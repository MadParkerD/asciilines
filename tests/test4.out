aktaaaaaaa
blubbbbbbb
cmvccccccc
dnwddddddd
enxeeeeeee
foyfffffff
gpzggggggg
hqzhhhhhhh
irziiiiiii
jszjjjjjjj
